spring-integration-mongodb/src/main/java/org/springframework/integration/mongodb/store/AbstractConfigurableMongoDbMessageStore.java

```diff
@@ -86,6 +87,8 @@ public abstract class AbstractConfigurableMongoDbMessageStore extends AbstractMe
 
 	private MessageBuilderFactory messageBuilderFactory = new DefaultMessageBuilderFactory();
 
+	private boolean createIndex = true;
```

Comment:
Didn't we agree that property name should be a `createIndexes` since we really might create several of them, not just one as it is assumed by this singular current name?